server/user.go

```diff
@@ -34,17 +34,17 @@ type Connection struct {
 	DefaultProjectKey  string `json:"default_project_key,omitempty"`
 }
 
-func (c *Connection) JiraAccountID() types.ID {
-	if c.AccountID != "" {
-		return types.ID(c.AccountID)
+func (connection *Connection) JiraAccountID() types.ID {
```

Comment:
why this change i think the previous change also looks good to me

Reply:
Because of this only i was getting lint error: `ST1016: methods on the same type should have the same receiver name (seen 1x "c", 1x "connection") (stylecheck)`